packages/react/src/elements/components/RichText/RichText.test.js

```diff
@@ -1,19 +0,0 @@
-import React from "react";
```

Comment:
Why was this removed? Did it start failing?

Reply:
This was removed in error. Thanks for catching.